foxai/explainer/computer_vision/model_utils.py

```diff
@@ -47,3 +48,34 @@ def get_last_conv_model_layer(model: ModelType) -> LayerType:
         raise ValueError("The model does not contain convolution layers.")
 
     return conv_layers[-1]
+
+
+def preprocess_baselines(
+    baselines: BaselineType,
+    input_data_shape: torch.Size,
+) -> Tuple[List[BaselineType], bool]:
+    """Adjust provided baselines to match input data dimension.
+
+    When baselines should have the same dimension as data samples or should have
+    additional first dimension to compute averaged attributes across all baselines.
+
+    Args:
+        baselines: Single tensor matching data sample dimensions or tensor
+            containing batch of baselines for each data sample.
+        input_data_shape: Shape of data sample batch.
+
+    Returns:
+        Tuple of baselines list and flag indicating if attributes should be
+            aggregated.
+    """
+    aggregate_attributes: bool = False
+    baselines_list: List[Union[None, torch.Tensor]] = [baselines]
```

Comment:
BaselineType to be more consistent.

Reply:
Fixed: https://github.com/softwaremill/FoXAI/pull/97/commits/e526164d4b0d6a6b4108a458a75bd9e6e2728776.